kefir.js.flow

```diff
@@ -0,0 +1,158 @@
+/* @flow */
+
+export type Event<V,E> =
+  {type: 'value', value: V} |
+  {type: 'error', value: E} |
+  {type: 'end', value: void};
+
+export type Emitter<V,E> = {
+  value(value: V): boolean;
+  event(event: Event<V,E>): boolean;
+  error(e: E): boolean;
+  end(): void;
+
+  // Deprecated methods
+  emit(value: V): boolean;
+  emitEvent(event: Event<V,E>): boolean;
+};
+
+export type Subscription = {
+  closed: boolean;
+  unsubscribe(): void;
+};
+
+export type Observer<V,E> = {
+  value(value: V): void;
+  error(err: E): void;
+  end(): void;
+};
+
+declare class Observable<+V,+E=*> {
+  toProperty(): Property<V,E>;
+  toProperty<V2>(getCurrent: ?() => V2): Property<V|V2,E>;
```

Comment:
Does it need to be optional since first overload covers the case when argument not passed?